recipes/glfw/all/patches/3.3.3-0002-Use-dependencies-from-Conan.patch

```diff
@@ -0,0 +1,38 @@
+From 4e7147b8445f8bfc85a4199fa0781708d4c9f19c Mon Sep 17 00:00:00 2001
+From: Jordan Williams <jordan@jwillikers.com>
+Date: Fri, 3 Nov 2023 13:46:24 -0500
+Subject: [PATCH] Use dependencies from Conan
+
+---
+ CMakeLists.txt | 8 ++------
+ 1 file changed, 2 insertions(+), 6 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 097e5931..4953299f 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -246,7 +246,7 @@ endif()
+ if (_GLFW_WAYLAND)
+ 
+     include(FindPkgConfig)
+-    pkg_check_modules(Wayland REQUIRED
++    pkg_check_modules(Wayland REQUIRED IMPORTED_TARGET
+         wayland-client>=0.2.7
+         wayland-cursor>=0.2.7
+         wayland-egl>=0.2.7
+@@ -254,11 +254,7 @@ if (_GLFW_WAYLAND)
+ 
+     list(APPEND glfw_PKG_DEPS "wayland-client")
+ 
+-    list(APPEND glfw_INCLUDE_DIRS "${Wayland_INCLUDE_DIRS}")
+-    list(APPEND glfw_LIBRARIES "${Wayland_LIBRARIES}" "${CMAKE_THREAD_LIBS_INIT}")
```

Comment:
It's possible to manipulate the CMakeDeps cmake file names. Does it help, instead of applying a new patch? 

https://docs.conan.io/2/reference/tools/cmake/cmakedeps.html#overwrite-properties-from-the-consumer-side-using-cmakedeps-set-property

Reply:
I forgot about that. I'll definitely give that a try.

Reply:
Yep, that allows me to drop an entire patch by tweaking the xkbcommon CMake config file name to be `XKBCommon`. This is done now.